src/sentry/static/sentry/app/views/organizationEvents/events.jsx

```diff
@@ -68,58 +63,37 @@ class OrganizationEvents extends AsyncView {
     return `Events - ${this.props.organization.slug}`;
   }
 
-  handleSearch = query => {
-    let {router, location} = this.props;
-    router.push({
-      pathname: location.pathname,
-      query: {
-        ...(location.query || {}),
-        query,
-      },
-    });
-  };
+  renderError() {
+    return this.renderBody();
+  }
+
+  renderLoading() {
+    return this.renderBody();
+  }
 
   renderBody() {
-    const {organization, location} = this.props;
-    const {reloading, events, eventsPageLinks} = this.state;
+    const {organization} = this.props;
+    const {loading, reloading, events, eventsPageLinks} = this.state;
 
     return (
       <React.Fragment>
-        <Flex align="center" justify="space-between" mb={2}>
-          <HeaderTitle>
-            {t('Events')} <BetaTag />
-          </HeaderTitle>
-          <StyledSearchBar
-            query={location.query && location.query.query}
-            placeholder={t('Search for events, users, tags, and everything else.')}
-            onSearch={this.handleSearch}
-          />
-        </Flex>
-
+        {this.state.error && super.renderError()}
```

Comment:
🤔

Reply:
`AsyncComponent.renderError` has behavior that I wanted to keep, but also wanted to render the existing body when there are errors.

Reply:
Yeah, it felt a little odd, but I don't have any better suggestion